OpenRA.Mods.Common/Traits/AffectsShroud.cs

```diff
@@ -86,7 +87,7 @@ PPos[] ProjectedCells(Actor self)
 				.ToArray();
 		}
 
-		void ITick.Tick(Actor self)
+		void INotifyVisualPositionChanged.VisualPositionChanged(Actor self, byte oldLayer, byte newLayer)
```

Comment:
This no longer updates if `Range` changes due to a condition or `IRevealsShroudModifier`/`ICreatesShroudModifier` from the subclasses.

I don't see a good alternative solution here other than keeping `ITick`.

Reply:
Fixed.